tests/symmetry/test_analyzer.py

```diff
@@ -574,8 +575,8 @@ def test_dihedral(self):
         assert pg_analyzer.sch_symbol == "Ih"
 
     def test_symmetrize_molecule1(self):
-        np.random.seed(77)
-        distortion = np.random.randn(len(C2H4), 3) / 10
+        rng = np.random.default_rng(77)
+        distortion = rng.standard_normal((len(C2H4), 3)) / 10
```

Comment:
For some reason the random distortion from the new Generator breaks the unit test (change the denominator to `20` for smaller distortion fixes this though, but need to confirm whether the tolerance is intended)

Reply:
oh yeah, i was actually going to ask if you updated the tests to work with new random behavior moving from std lib `random` to `np.random`. i think it's fine for a test to depend on the right random seed. can you update the tests?

Reply:
> oh yeah, i was actually going to ask if you updated the tests to work with new random behavior moving from std lib `random` to `np.random`. 

Okay I would do that very soon :)